plugins/modules/cloud/misc/terraform.py

```diff
@@ -95,6 +95,11 @@
     description:
       - A group of key-values to provide at init stage to the -backend-config parameter.
     required: false
+  backend_config_file:
```

Comment:
Why not call this option `backend_config_files`?

Reply:
I just wanted to keep compliance with the former option format `variables_file`. I have also changed that option to accept lists, but keep the name and didn't change that to `variables_files` to keep backward compatibility. Shall I change this new one to  `backend_config_files` or keep the format same as the other option. @felixfontein

Reply:
I would rename the new one, and add an alias to the old one.

Reply:
done